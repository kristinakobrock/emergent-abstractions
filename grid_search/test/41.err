  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  8%|#######4                                                                                     | 4/50 [00:00<00:01, 29.84it/s] 14%|#############                                                                                | 7/50 [00:00<00:01, 24.28it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 25.98it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 25.71it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 26.47it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 20.79it/s] 48%|############################################1                                               | 24/50 [00:01<00:01, 22.01it/s] 54%|#################################################6                                          | 27/50 [00:01<00:01, 22.88it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 23.51it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 24.05it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 26.93it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 26.49it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 26.08it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 25.95it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 24.79it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.80it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 29.99it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 24.99it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 23.50it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 24.26it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.15it/s]
