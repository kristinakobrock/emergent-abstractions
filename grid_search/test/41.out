# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6802768111228943, "acc": 0.5981299877166748, "length": 3.5642008781433105, "mode": "train", "epoch": 1}
{"loss": 0.6552389860153198, "acc": 0.6398437023162842, "length": 4.0, "mode": "test", "epoch": 1}
{"loss": 0.6680974364280701, "acc": 0.6138113737106323, "length": 3.594834804534912, "mode": "train", "epoch": 2}
{"loss": 0.6559078693389893, "acc": 0.6402344107627869, "length": 4.0, "mode": "test", "epoch": 2}
{"loss": 0.6642719507217407, "acc": 0.6255509257316589, "length": 3.612060546875, "mode": "train", "epoch": 3}
{"loss": 0.6519403457641602, "acc": 0.6392578482627869, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.6622682213783264, "acc": 0.6252268552780151, "length": 3.6517722606658936, "mode": "train", "epoch": 4}
{"loss": 0.6536655426025391, "acc": 0.6400390267372131, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.6606488227844238, "acc": 0.6253731846809387, "length": 3.6747496128082275, "mode": "train", "epoch": 5}
{"loss": 0.6517810821533203, "acc": 0.6404297351837158, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.659511148929596, "acc": 0.6254166960716248, "length": 3.69433331489563, "mode": "train", "epoch": 6}
{"loss": 0.6497669219970703, "acc": 0.6416015625, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.6589155793190002, "acc": 0.6262796521186829, "length": 3.7314112186431885, "mode": "train", "epoch": 7}
{"loss": 0.6415885090827942, "acc": 0.6503906846046448, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.6546990275382996, "acc": 0.6277981400489807, "length": 3.784045934677124, "mode": "train", "epoch": 8}
{"loss": 0.6314049959182739, "acc": 0.6566405892372131, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.6478541493415833, "acc": 0.6352717280387878, "length": 3.821268081665039, "mode": "train", "epoch": 9}
{"loss": 0.6310871839523315, "acc": 0.664257824420929, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.6377313733100891, "acc": 0.6420787572860718, "length": 3.858640193939209, "mode": "train", "epoch": 10}
{"loss": 0.6286282539367676, "acc": 0.6529297232627869, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.6256304979324341, "acc": 0.6588383316993713, "length": 3.889556884765625, "mode": "train", "epoch": 11}
{"loss": 0.6174476146697998, "acc": 0.6605468392372131, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.6150898337364197, "acc": 0.670678973197937, "length": 3.9214158058166504, "mode": "train", "epoch": 12}
{"loss": 0.6237112283706665, "acc": 0.6507812142372131, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.6061362624168396, "acc": 0.6701785326004028, "length": 3.942633867263794, "mode": "train", "epoch": 13}
{"loss": 0.6330230236053467, "acc": 0.6500000357627869, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.6033281683921814, "acc": 0.6737789511680603, "length": 3.9581499099731445, "mode": "train", "epoch": 14}
{"loss": 0.6336536407470703, "acc": 0.6478515267372131, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5981931090354919, "acc": 0.680048406124115, "length": 3.9714372158050537, "mode": "train", "epoch": 15}
{"loss": 0.6512637138366699, "acc": 0.6373047232627869, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5948778986930847, "acc": 0.6808353066444397, "length": 3.9807159900665283, "mode": "train", "epoch": 16}
{"loss": 0.6490480899810791, "acc": 0.6429687142372131, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.5923550724983215, "acc": 0.685370922088623, "length": 3.9823572635650635, "mode": "train", "epoch": 17}
{"loss": 0.6610149145126343, "acc": 0.6419921517372131, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5899584889411926, "acc": 0.6865190863609314, "length": 3.98404598236084, "mode": "train", "epoch": 18}
{"loss": 0.6497195959091187, "acc": 0.6539062857627869, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.5863366723060608, "acc": 0.6877240538597107, "length": 3.9874653816223145, "mode": "train", "epoch": 19}
{"loss": 0.6348029971122742, "acc": 0.6603515148162842, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.5854654312133789, "acc": 0.6895040273666382, "length": 3.9876744747161865, "mode": "train", "epoch": 20}
{"loss": 0.6281581521034241, "acc": 0.659960925579071, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.5816695094108582, "acc": 0.6940186619758606, "length": 3.99029278755188, "mode": "train", "epoch": 21}
{"loss": 0.6199280619621277, "acc": 0.6630859971046448, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.5789162516593933, "acc": 0.6922516226768494, "length": 3.9902896881103516, "mode": "train", "epoch": 22}
{"loss": 0.6128547191619873, "acc": 0.6708984375, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.5759512782096863, "acc": 0.696945071220398, "length": 3.9900691509246826, "mode": "train", "epoch": 23}
{"loss": 0.60765141248703, "acc": 0.6746094226837158, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.5709155797958374, "acc": 0.7022039890289307, "length": 3.9930148124694824, "mode": "train", "epoch": 24}
{"loss": 0.6068479418754578, "acc": 0.6810545921325684, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.5659872889518738, "acc": 0.7049932479858398, "length": 3.991824150085449, "mode": "train", "epoch": 25}
{"loss": 0.6042200326919556, "acc": 0.6826171875, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.5628581643104553, "acc": 0.7094747424125671, "length": 3.9925549030303955, "mode": "train", "epoch": 26}
{"loss": 0.597794234752655, "acc": 0.6824217438697815, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.5571489930152893, "acc": 0.711739182472229, "length": 3.992690324783325, "mode": "train", "epoch": 27}
{"loss": 0.590667724609375, "acc": 0.6873047351837158, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.5512517094612122, "acc": 0.7165734171867371, "length": 3.993259906768799, "mode": "train", "epoch": 28}
{"loss": 0.591575026512146, "acc": 0.6921874284744263, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.5471572279930115, "acc": 0.7171714305877686, "length": 3.992779016494751, "mode": "train", "epoch": 29}
{"loss": 0.6043922901153564, "acc": 0.6759765148162842, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.5457431674003601, "acc": 0.7212212085723877, "length": 3.9922103881835938, "mode": "train", "epoch": 30}
{"loss": 0.5946841835975647, "acc": 0.6880859732627869, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.5422196984291077, "acc": 0.721950888633728, "length": 3.9939918518066406, "mode": "train", "epoch": 31}
{"loss": 0.59308922290802, "acc": 0.6896485090255737, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.5389105081558228, "acc": 0.724676251411438, "length": 3.9931704998016357, "mode": "train", "epoch": 32}
{"loss": 0.5867119431495667, "acc": 0.6919922232627869, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.5351618528366089, "acc": 0.7302806973457336, "length": 3.993546724319458, "mode": "train", "epoch": 33}
{"loss": 0.5846505165100098, "acc": 0.6994141936302185, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.5326387286186218, "acc": 0.7306459546089172, "length": 3.9920082092285156, "mode": "train", "epoch": 34}
{"loss": 0.602726936340332, "acc": 0.6955079436302185, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.5289222002029419, "acc": 0.7330136895179749, "length": 3.9902756214141846, "mode": "train", "epoch": 35}
{"loss": 0.5764797925949097, "acc": 0.7078125476837158, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.5230643153190613, "acc": 0.7377694845199585, "length": 3.9904446601867676, "mode": "train", "epoch": 36}
{"loss": 0.5736539363861084, "acc": 0.7175781726837158, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.5179751515388489, "acc": 0.7443938851356506, "length": 3.9914658069610596, "mode": "train", "epoch": 37}
{"loss": 0.5680272579193115, "acc": 0.7251953482627869, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.5123867988586426, "acc": 0.7503803968429565, "length": 3.9911201000213623, "mode": "train", "epoch": 38}
{"loss": 0.5745234489440918, "acc": 0.73046875, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.507837176322937, "acc": 0.7602701187133789, "length": 3.991672992706299, "mode": "train", "epoch": 39}
{"loss": 0.5517507791519165, "acc": 0.7359375357627869, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.49858203530311584, "acc": 0.7642021179199219, "length": 3.9922780990600586, "mode": "train", "epoch": 40}
{"loss": 0.5421628355979919, "acc": 0.7468749284744263, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.4879154562950134, "acc": 0.7724697589874268, "length": 3.991780996322632, "mode": "train", "epoch": 41}
{"loss": 0.5404220819473267, "acc": 0.7490234971046448, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.48029080033302307, "acc": 0.7795875668525696, "length": 3.9931931495666504, "mode": "train", "epoch": 42}
{"loss": 0.5292103290557861, "acc": 0.7589843273162842, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.46979793906211853, "acc": 0.7808057069778442, "length": 3.9947993755340576, "mode": "train", "epoch": 43}
{"loss": 0.5268797874450684, "acc": 0.7619141936302185, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.4674503207206726, "acc": 0.7864070534706116, "length": 3.994753360748291, "mode": "train", "epoch": 44}
{"loss": 0.5238295197486877, "acc": 0.7679687142372131, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.45666974782943726, "acc": 0.7902036309242249, "length": 3.994227409362793, "mode": "train", "epoch": 45}
{"loss": 0.5246614217758179, "acc": 0.7687499523162842, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.4526735842227936, "acc": 0.793267011642456, "length": 3.9948477745056152, "mode": "train", "epoch": 46}
{"loss": 0.5211646556854248, "acc": 0.7712889909744263, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.447494238615036, "acc": 0.7988369464874268, "length": 3.9944279193878174, "mode": "train", "epoch": 47}
{"loss": 0.5107822418212891, "acc": 0.7767577767372131, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.4419395327568054, "acc": 0.8043263554573059, "length": 3.994354248046875, "mode": "train", "epoch": 48}
{"loss": 0.5179274678230286, "acc": 0.7744141221046448, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.44003263115882874, "acc": 0.8050656318664551, "length": 3.9949021339416504, "mode": "train", "epoch": 49}
{"loss": 0.5094295740127563, "acc": 0.7794921398162842, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.4348808228969574, "acc": 0.8094655275344849, "length": 3.99504017829895, "mode": "train", "epoch": 50}
{"loss": 0.5115181803703308, "acc": 0.7808594107627869, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.4273953139781952, "acc": 0.8132846355438232, "length": 3.995417594909668, "mode": "train", "epoch": 51}
{"loss": 0.5194867849349976, "acc": 0.7734375, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.4267280399799347, "acc": 0.8144097924232483, "length": 3.9957427978515625, "mode": "train", "epoch": 52}
{"loss": 0.5239951610565186, "acc": 0.7697266340255737, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.4238904118537903, "acc": 0.8136427998542786, "length": 3.9952380657196045, "mode": "train", "epoch": 53}
{"loss": 0.5007478594779968, "acc": 0.7832030653953552, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.4207333028316498, "acc": 0.8168118000030518, "length": 3.9963624477386475, "mode": "train", "epoch": 54}
{"loss": 0.48575472831726074, "acc": 0.7876953482627869, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.417344868183136, "acc": 0.8175265789031982, "length": 3.996290922164917, "mode": "train", "epoch": 55}
{"loss": 0.48619043827056885, "acc": 0.7867187857627869, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.4106830954551697, "acc": 0.8254463076591492, "length": 3.996525526046753, "mode": "train", "epoch": 56}
{"loss": 0.48476147651672363, "acc": 0.7908203601837158, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.4069161117076874, "acc": 0.8235482573509216, "length": 3.996591329574585, "mode": "train", "epoch": 57}
{"loss": 0.4872680902481079, "acc": 0.7794922590255737, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.4091300368309021, "acc": 0.820763885974884, "length": 3.9968039989471436, "mode": "train", "epoch": 58}
{"loss": 0.48086830973625183, "acc": 0.7859376668930054, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.40694886445999146, "acc": 0.823311448097229, "length": 3.9968276023864746, "mode": "train", "epoch": 59}
{"loss": 0.4747943878173828, "acc": 0.7908202409744263, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.39988014101982117, "acc": 0.8238145112991333, "length": 3.9981677532196045, "mode": "train", "epoch": 60}
{"loss": 0.46810996532440186, "acc": 0.7945312857627869, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7125000357627869
